huaweicloud/services/css/resource_huaweicloud_css_cluster.go

```diff
@@ -64,6 +66,8 @@ const (
 // @API CSS POST /v1.0/{project_id}/clusters/{cluster_id}/public/bandwidth
 // @API CSS PUT /v1.0/{project_id}/clusters/{cluster_id}/public/close
 // @API CSS PUT /v1.0/{project_id}/clusters/{cluster_id}/publickibana/close
+// @API CSS GET /v1.0/{project_id}/es-flavors
+// @API CSS POST /v1.0/{project_id}/clusters/{cluster_id}/{types}/flavor
```

Comment:
miss API of bss

Reply:
API comment of BSS already exists, no need to add it.